content/blog/pycon-india-2018.rst

```diff
@@ -4,6 +4,8 @@ Announcing PyCon India 2018!
 :Date: 2017-12-07 20:30
 :Category: 2018
 
+**New venue announced. Kindly have a look at this** `blog post <https://in.pycon.org/blog/2018/pycon-india-2018-reschedule.html>`_.
```

Comment:
Please use this text instead `This blog post is old. There has been a change in PyCon 2018 Location. The Announcement can be found in this`

Reply:
I'm on it! 👍